site/src/system/styles/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './reset.css';
```

Comment:
I'd honestly prefer not to have barrel files. They just cause indirection and don't serve much of a purpose in a relative small app like this docs site.